blocks/linkable-cards/linkable-cards.css

```
@@ -66,12 +66,26 @@
   box-shadow: 2px 4px 4px 2px rgb(0 0 0 / 15%);
 }
 
+.linkable-cards.purple-cards > div {
+  background-color: #6400A6;
```

Let's use a CSS variable for this hex code. You can add this `purple-140` to the main stylesheet. (https://github.com/hlxsites/merative2/blob/main/styles/styles.css#L93) 

`--purple-140: #6400A6;`


```suggestion
  background-color: var(--purple-140);
```

---

Done